app/components/modals/add-user-role-modal.js

```diff
@@ -0,0 +1,53 @@
+import FormMixin from 'open-event-frontend/mixins/form';
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
+
+export default ModalBase.extend(FormMixin, {
+  isSmall            : true,
+  autoScrollToErrors : false,
+
+  actions: {
+    addRole() {
```

Comment:
@niranjan94 I have left some comments on the issue #302 , regarding why this action is here <temporarily>
Please guide